chingu-voyages/v37-bears-team-14
# Add tech stack update modal to project page

# Changes

- Adds a modal for project owners to update the tech stack of a project

<img width="879" alt="Screen Shot 2022-02-24 at 11 23 26 PM" src="https://user-images.githubusercontent.com/8432061/155672661-9f2deda2-6146-4a19-9868-8a36062e84d0.png">

# Testing
- Run app
- Try adding a tech stack item to a project that your user owns (should succeed)
- Try adding a tech stack item to a project you are not an owner of (should fail)


Closes #13
